### DISCUSSION TITLE
There's a new windows update so...

### DISCUSSION THREAD

**Discussion by blecaplol** [valinet/ExplorerPatcher#1553] [Q&A]
Am I safe to update it?  
https://gyazo.com/92ba1f99cb447000d1f23c9985beaf12

**Comment by evoIutionevotv** [comment]
i would be careful, last time i updated with explorerpatcher my entire explorer broke

**Comment by pyrates999** [comment]
This looks like spam with seeing a random link.

**Reply by blecaplol** [reply]
then don't click? lmao.
